.pre-commit-config.yaml

```diff
@@ -10,6 +10,9 @@ repos:
           --ignore=A002,
           --ignore=D101,
           --ignore=D102,
+          --ignore=ARG001,
+          --ignore=F841,
+          --ignore=FIX002,
           --ignore=PLR0913,
```

Comment:
This is:
- TODO
- Unused parameter
- unused local variable

Reply:
Could you add these human readable explanations as comments to every line of these? (also the ones that existed before?)